pkgcfg/pkgcfg.go

```diff
@@ -4,16 +4,21 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"io/fs"
 	"os"
 	"path/filepath"
+	"strings"
 	"text/template"
 
 	"github.com/pkg/errors"
 	"go.jetpack.io/devbox/debug"
 	"go.jetpack.io/devbox/nix"
 )
 
-const confPath = ".devbox/conf"
+const (
+	ConfBinPath = ".devbox/conf/bin"
+	confPath    = ".devbox/conf"
```

Comment:
Same question - should these still have `.devbox/`?

Reply:
Same comment here, this is correct but confusing because now we have 2 conf directories (one hidden and one not). Maybe we do:

* `./config` is for configs  (or maybe `./plugins`) ? 
* `.devbox/plugin-data/`  for hidden non-permanent non-shared data.